django_ipam/base/admin.py

```diff
@@ -22,6 +22,8 @@ class AbstractSubnetAdmin(TimeReadonlyAdminMixin, ModelAdmin):
     change_form_template = "admin/django-ipam/subnet/change_form.html"
     change_list_template = "admin/django-ipam/subnet/change_list.html"
     app_name = "django_ipam"
+    list_display = ('subnet', 'name', 'master_subnet', 'description')
+    search_fields = ['subnet', 'name']
```

Comment:
One question: does `name` have `db_index=True` in the model? If not please add it, it will speed up the search;
it makes sense to search by name so adding a database index is a very reasonable choice.
Do this change in a new commit on this same PR please.